odinApp/src/odinRestApi.h

```diff
@@ -5,12 +5,16 @@
 #include <string>
 #include <epicsMutex.h>
 #include <osiSock.h>
+#include <cstdint>
```

Comment:
Is this a C++11 thing? There is still a std=c++0x flag in the src Makefile that I want to take out but this requires it. It seems to build OK if I remove it, though.

Reply:
Mmm not sure why I added that specifically.  I must have had a problem with includes and perhaps added this in error.  If it works without it then get rid.